src/telemetry.rs

```diff
@@ -47,6 +47,7 @@ impl Zone {
     }
 }
 
+#[derive(Clone, Debug)]
```

Comment:
I am not quite sure if cloning guard struct is useful?

Reply:
Reverted this one too.